feature-utils/poly-look/test/react-components/filterChips.test.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { render, fireEvent, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import FilterChips from "../../src/react-components/filterChips.jsx";
+
+const chipsContent = [...Array(3).keys()].map((i) => `chip${i}`);
+const chipsContentObject = [...Array(3).keys()].map((i) => {
+  return {
+    id: `chip${i}`,
+  };
+});
+const mockedHandleClick = jest.fn();
+const lightTheme = "light";
+const darkTheme = "dark";
+
+const component = (
+  <FilterChips
+    chipsContent={chipsContent}
+    onChipClick={mockedHandleClick}
+    defaultActiveChips={[]}
+    theme
+    exclusive
+  />
+);
+
+it("Chips content is an object array", () => {
+  render(<FilterChips chipsContent={chipsContentObject} />);
+  for (const chipId of chipsContent) {
+    const chipElement = screen.getByText(chipId);
+    expect(chipElement).toBeInTheDocument();
+  }
+});
+
+describe("Chips content is an array", () => {
+  beforeEach(() => {
+    render(component);
+  });
+
+  it("Chips present in document", () => {
+    for (const chipId of chipsContent) {
+      const chipElement = screen.getByText(chipId);
+      expect(chipElement).toBeInTheDocument();
+    }
+  });
+
+  describe("Checks onChipClick event", () => {
```

Comment:
You can nest `describes`, it's not syntactically incorrect, but it's probably better if you just use a single level. In this case you don't have any fixtures, you use describe just for grouping, so it might be better to just include the describe string into the `it` description.